### PR TITLE
fix: restructured some files, functions & folders.

### PR DESCRIPTION
**Pull request by bharathraj25**
* Moved some functions to utils file.
* removed destructing in PO Note controllers file, required for testing.
* Removed some unwanted folder & files.
* Overall refactored functions & files.

### CODE REVIEW

**Review by Harshit-Soni-Mck — COMMENTED** [review]
add error handlers for each call or create a interceptor that can automatically handle errors

File: src/services/poNoteServices.js [hunk 1]
```diff
@@ -116,22 +81,15 @@ const createValidPONote = async (
...
     note,
-    ...(status && { status })
+    ...(status && { status }),
+    ...(dueDate && {
+      dueDate: new Date(dueDate).toISOString()
+    }),
+    ...(issueLink && { issueLink })
   };
 
   const createdNote = await prisma.PONote.create({
```
[the remark this answers is not in the document]
Comment (bharathraj25) [reply src/services/poNoteServices.js:91]: Yes, error handling is added as middle ware using joi validation.

**Comment by bharathraj25** [comment]
> add error handlers for each call or create a interceptor that can automatically handle errors

Yes, error handling is added as middle ware using joi validation.

**Review by sugumar-mani — APPROVED** [review]
Loogk ok to me. Please check with @Harshit-Soni-Mck  and add the fix if needed in this PR or next

**Pull request merged**